### ISSUE TITLE
[ENHANCEMENT] Formulation of HantushWellModel

### ISSUE THREAD

**Issue by bdestombe** [pastas/pastas#605]
**Describe the proposed enhancement**
Why is the HantushWellModel not formulated such that parameter A represents the T (transmissivity)? This would allow for direct interpretation of the calibrated value and its uncertainty.

**Comment by rubencalje** [comment]
See issue #210

**Comment by bdestombe** [comment]
> See issue #210

Thanks Ruben for your reply. That referred issue discusses parameter a and the complications related to changing parameters, not so much parameter A. Or am I missing something?

**Comment by dbrakenhoff** [comment]
Hi @bdestombe, I'm guessing you're basing your question on the formulas shown in [this notebook](https://pastas.readthedocs.io/en/latest/benchmarks/check_response_functions.html)? Parameter `A'` is shown to equal:

`A' = - 1 / (2 * np.pi * T)`

Which means we could just implement `A' = T` and convert the internal equation accordingly.

Two bad reasons we haven't done so:
- The current implementation was based on the original `Hantush` implementation, so we kept it somewhat similar. 
- The math is pretty simple if you want to make this translation yourself, also for the uncertainty :innocent:? 

But one thing I do wonder about: 

I'm not sure we want to necessarily encourage the interpretation of a parameter as a physical parameter. That's not to say that it isn't interesting in certain cases (it definitely is!, for example when the system you're modeling is similar to the Hantush conceptual model). But in many cases the value might not make physical sense (and that is okay), since we're not actually modeling an aquifer, but using the Hantush formula as a blueprint for the response of a system to a pumping well. And even in systems that are similar to the Hantush conceptual model you might not be able to get a reasonable estimate of parameter `A'` or `T` because you can really only get good estimates of combinations of aquifer parameters. For `HantushWellModel` I have found that the uncertainty of `A'` and `b'` individually can be relatively high, whereas the uncertainty of the gain can be relatively low. This shows that the gain can be estimated well, but the individual parameters not so much. But this is true regardless of which definition of `A'` we implement. 

Long story short, I'll include your question in our next developers meeting. I'll keep you posted on what we decide!

**Comment by bdestombe** [comment]
Thank you so much for your extended answer!

And I completely agree that you could get a perfect fit for the wrong reasons. In line with your reasoning you could also argue that if the fit is nice and the transmissivity is within the range of expectation, I could "trust" that the Hantush model structure to be representative of the real setting. If the value is outside of the expected range, it could still be a helpfull exercise. 

The uncertainty in that gainfactor that you discuss at the end is a great find! Most likely because most of the time and most of the measurements the system is in or close to steady state, making it easier to estimate? And definitely easier to find an initial estimate. But this gainfactor is for the Hantus function, for the HantushWellModel something else could be picked..

Thanks for your response again.
